yarn.lock

```diff
@@ -149,14 +149,6 @@
   dependencies:
```

Comment:
REVIEWERS: how do I revert this? I'm not sure if this file is supposed to be changed. Let me know your thoughts.